File: Reverse Words in a String III.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    let st = '';
    let sp = s.split(" ");
    for (let i = 0; i < sp.length; i++) {
        sp[i] = sp[i].split('').reverse().join('');
        st += sp[i];
        if (i < sp.length - 1) {
            st += " ";
        }
    }
    return st;
};

File: Reverse Words in a String III.py
class Solution:
    def reverseWords(self, s: str) -> str:
        st = ''
        sp = s.split(" ")
        for i in range(len(sp)):
            sp[i] = sp[i][::-1]
            st += sp[i]
            if i < len(sp) - 1:
                st += " "
        return st
